components/Fundraise/FundraiseCard.tsx

```diff
@@ -0,0 +1,273 @@
+import React, { ReactElement, useMemo } from "react";
+import { Fundraise, isFundraiseFulfilled } from "./lib/types";
+import { css, StyleSheet } from "aphrodite";
+import numeral from "numeral";
+import colors from "~/config/themes/colors";
+import { timeTo } from "~/config/utils/dates";
+import CommentAvatars from "../Comment/CommentAvatars";
+import ResearchCoinIcon from "../Icons/ResearchCoinIcon";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCircleCheck } from "@fortawesome/pro-solid-svg-icons";
+import { faClock } from "@fortawesome/pro-regular-svg-icons";
+import FundraiseContributorsModal from "./ContributorsModal";
+import Button from "../Form/Button";
+import FundraiseContributeModal from "./ContributeModal";
+import PermissionNotificationWrapper from "../PermissionNotificationWrapper";
+import ContentBadge from "../ContentBadge";
+import { formatBountyAmount } from "~/config/types/bounty";
+import { breakpoints } from "~/config/themes/screen";
+
+export type FundraiseCardProps = {
+  fundraise: Fundraise;
+
+  showSupportButton?: boolean;
+  onUpdateFundraise?: (fundraise: Fundraise) => void;
+};
+
+const FundraiseCard = ({
+  fundraise: f,
+  showSupportButton = true,
+  onUpdateFundraise = () => {},
+}: FundraiseCardProps): ReactElement => {
+  const barFillAmount = useMemo(() => {
+    const fillAmount = (f.amount_raised.rsc / f.goal_amount.rsc) * 100;
```

Comment:
If you'd like to one-line it you can: `Math.min((f.amount_raised.rsc / f.goal_amount.rsc) * 100, 100)`.
Fine as is though.